.gitattributes

```diff
@@ -1,3 +1,6 @@
 /.gitattributes export-ignore
 /.gitignore export-ignore
 /.github export-ignore
+/CODE_OF_CONDUCT.md export-ignore
+/README.org export-ignore
+/CREDITS export-ignore
```

Comment:
Similarly, for copyright reasons, I think it would be better to include the `CREDITS` file.